build-support/pants_venv

```diff
@@ -83,7 +83,7 @@ function activate_pants_venv() {
     ensure_gcc
 
     for req in "${REQUIREMENTS[@]}"; do
-      pip install ${pip_extra} -r "${req}" || \
+      pip install "${pip_extra}" -r "${req}" || \
```

Comment:
Huh. It looks like this change breaks CI: https://travis-ci.com/github/pantsbuild/pants/jobs/322681522#L647. Will need to revert.

Reply:
pre commit hook ask me to change this place, it said `shellcheck` not happy with it

Reply:
I revert it and now looks good!